src/lifecycle/created.js

```diff
@@ -43,6 +43,18 @@ function callCreatedOnDescendants (elem, id) {
   });
 }
 
+function createCallUpdateOnProperties (opts) {
+  let props = opts.properties || {};
+  let names = Object.keys(props);
+  return function (elem) {
+    names.forEach(function (name) {
+      let prop = props[name];
+      let update = prop && prop.update;
+      update && update.call(elem, elem[name]);
```

Comment:
could you combine these two lines?
`prop && prop.update && prop.update.call(elem, elem[name]);`